website/docs/cdktf/concepts/assets.mdx

```diff
@@ -22,19 +22,25 @@ The TypeScript example below uses `TerraformAsset` to upload the contents of the
 
 The stack output directory in `cdktf.out` contains all of the assets that `TerraformAsset` needs. This is important for workflows where you use synthesized configurations with Terraform directly. For example, you would only need to upload the contents of the stack output folder to Terraform Cloud or Terraform Enterprise.
 
+<!-- #NEXT_CODE_BLOCK_SOURCE:ts examples/typescript/documentation#assets -->
+<!-- #xxNEXT_CODE_BLOCK_SOURCE:python examples/python/documentation#assets -->
```

Comment:
Maybe we can use the convention over configuration paradigm here and reduce it to `xxNEXT_CODE_BLOCK_SOURCE:python assets` and let the rest be defaulted

Reply:
I'd rather be explicit here because it helps users / future me when reading our Markdown source where to find this snippet and it also helps us to find it quicker.
E.g. if I'd search for `DOCS_BLOCK_START:constructs` I'd miss `DOCS_BLOCK_START:assets,constructs` when trying to find this. Searching for just "constructs" or "assets" does not really help 😅